### DISCUSSION TITLE
Example of the initial steps to obtain an access token

### DISCUSSION THREAD

**Discussion by edevil** [lexmag/oauther#27] [Q&A] [answered]
It would be great if we could also have examples of how to obtain the request token, redirect to the authorize url, and then obtaining the access token to access the protected resources.

**Comment by jordpo** [comment]
@edevil were you able to get an access token successfully? I'm signing the request but the signature is always invalid - `OAuther.sign(verb, url, [{"oauth_verifier", oauth_verifier}, {"oauth_token", oauth_token}], credentials)`. The credentials are fine - they are the same that work for the request token. Am I missing another header?

**Comment by jordpo** [comment, marked as answer]
Yep, the secret and request_token should be used in the credentials. The only thing that you should add to the sign call is the oauth_verifier.
